providers/implementations/digests/blake2b_prov.c

```diff
@@ -17,9 +17,49 @@
 #include <assert.h>
 #include <string.h>
 #include <openssl/crypto.h>
+#include <openssl/core_names.h>
+#include <openssl/proverr.h>
+#include <openssl/err.h>
 #include "blake2_impl.h"
 #include "prov/blake2.h"
 
+static const OSSL_PARAM known_blake2b_settable_ctx_params[] = {
+    {OSSL_DIGEST_PARAM_XOFLEN, OSSL_PARAM_UNSIGNED_INTEGER, NULL, 0, 0},
+    OSSL_PARAM_END
+
+};
```

Comment:
I'd like to see at least one or two testcases in blake2b evp test file that will test the XOFLEN parameter.

Reply:
https://github.com/openssl/openssl/pull/12256/commits/20f84d7f8d6d08393104e3246aa637a4ec5e5ac8 https://github.com/openssl/openssl/pull/12256/commits/d7425e74e904868de357720a4f6188fadf1b42ef